sensing/src/main/java/com/google/android/sensing/upload/SensorDataUploadWorker.kt

```diff
@@ -53,7 +48,7 @@ class SensorDataUploadWorker(appContext: Context, workerParams: WorkerParameters
 
   override suspend fun doWork(): Result {
     if (!tryAcquiringLock()) {
-      return Result.success(workDataOf("State" to SyncUploadState.NoOp::class.java))
+      return Result.retry()
```

Comment:
Why is retry made when the lock is not acquired?

Reply:
I was debugging something. Forgot to change back.
Thanks. Done!